src/lib/components/user/Profile.svelte

```diff
@@ -7,51 +7,41 @@
 	import type { User } from '@prisma/client';
 	import RichViewer from '../editor/RichViewer.svelte';
 	import { formatRelative } from 'date-fns';
+	import Gravatar from './Gravatar.svelte';
 
 	export let user: User;
 
 	let userCountry: string = user.country || '';
 	let userCountryName: string = countries.find((c) => c.code == userCountry).name || '';
+	let memberSince = formatRelative(new Date(user.createdAt), new Date());
 </script>
 
 <div class="container-xxl mt-4">
-	<div class="row">
-		<div class="col-2 my-4">
-			{#if user.id === $currentUser?.id}
-				<div class="row">
-					<a href={`${base}/user/profile/edit`}>Edit Profile</a>
-				</div>
-			{/if}
+	<div class="row my-4">
+		<div class="col-1 pe-0">
+			<Gravatar email={'jwaterfaucett@gmail.com'} />
```

Comment:
emails should be on the user object, we can add either a helper function in lib/state/user.ts or just here that does something like `user.emails.find(e => e.primary && e.confirmed)`